eva/server/interpreter.py

```diff
@@ -56,9 +56,8 @@ async def read_from_client_and_send_to_server(
     stdin_reader: StreamReader, writer: StreamWriter, server_reader: StreamReader
 ):
     VERSION = VERSION_DICT["VERSION"]
-    intro = "eva (v " + VERSION + ')\nType "EXIT;" to exit the client' + "\n"
-    sys.stdout.write(intro)
-    sys.stdout.flush()
+    intro = f"eva (v{VERSION})\nType 'EXIT;' to exit the client \n"
+    print(intro, flush=True)
```

Comment:
Is there a reason for using `print` instead of logger or sys.stdout here?

Reply:
`print` is just a wrapper around `sys.stdout`.